examples/kafka/Dockerfile.example.kafka

```diff
@@ -0,0 +1,14 @@
+FROM pai.build.base:hadoop2.7.2-cuda8.0-cudnn6-devel-ubuntu16.04
+
+#install python3 and vim
+RUN apt-get update && apt-get install python3.4 && apt-get install vim
+
+#install python-kafka(Kafka Python SDK)
+RUN pip install kafka-python
+
+WORKDIR /root
+#download zookeeper
```

Comment:
>zookeeper [](start = 10, length = 9)

Is it an official site to download the package?

Reply:
Yes, I get the URL from apache official site.